src/layouts/docs/utils/product-url-adjusters.ts

```diff
@@ -0,0 +1,26 @@
+import { RemarkPlugin, RemarkPluginOptions } from 'types/remark'
+import adjustLinkUrlPlugin from './adjust-link-urls-plugin'
+
+const vaultUrlAdjuster: [RemarkPlugin, RemarkPluginOptions] = [
+  adjustLinkUrlPlugin,
```

Comment:
Leverages the `adjustLinkUrlPlugin` to fix the `/api` link issue. We could, if we want, modify this to also handle the `href` adjustment currently happening in `DocsAnchor` - though the latter is already working fine, so might be preferable to leave that question out of this PR.